src/components/Comment.tsx

```diff
@@ -0,0 +1,67 @@
+import React from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import {
+    Box,
+    Button,
+    Container,
+    Grid,
+    Typography
+} from '@material-ui/core';
+
+const useStyles = makeStyles({
+    root: {
+        borderStyle: 'solid'
+    },
+
+    date: {
+        float: 'left',
+        textAlign: 'left',
+        fontSize: '18px',
+        marginTop: '0.25rem',
+    },
+
+    content: {
+        marginTop: '0.75rem'
+    },
+    
+    deleteComment: {
+        paddingLeft: '2rem',
+        paddingRight: '2rem',
+        float: 'right',
+        textAlign: 'right',
+        background: 'lightgrey'
+    }
+
+});
+
+interface CommentProps {
+    username: string;
+    date: string;
+    claps: number;
+    content: string;
+}
+
+const Comment: React.FC<CommentProps> = ({username, date, claps, content}) => {
+    const classes = useStyles();
+
+    return(
+        <Container>
+            <Grid container className={classes.root}>
+                <Grid item xs={7}>
+                    <Typography variant="h6">{username} <Box component="span" fontSize="24px">👏</Box>  {claps}</Typography>
```

Comment:
Well this is certainly a first. I'm gonna lean towards not committing emojis into the codebase. Can we use an SVG?

Reply:
I thought the same, but Ryan suggested using unicode because it looked better

Reply:
@rymaju please elaborate on this, I haven't encountered it before.

Reply:
1. it just looked a bit better than the previous svg. 

2. My concern is that teaching people how to import SVG as a react component is actually a bit complicated (or worse, magical) and idk if you want to include that in the jumpstart frontend. Just having plain text unicode emoji makes sense and vibes with the kind of styling that they are probably already familiar with. @jackblanc